Automattic/simplenote-electron
# Add a title bar space for Mac Electron

### Fix

Previously we had been adding specific styles to different components for only Mac Electron app to prevent things from overlapping the window buttons.

This removes those and instead enables the default title bar.

To have this work well I also updated the Electron dependency to fix some issues with the rounded corners. With that also updated Electron Builder.

Resolves #2886

The Electron update introduced two issues one where logging out wouldn't refresh to the login screen. The other the Linux build wouldn't render properly.
Both of those have fixes here as well.

<details>
<summary>Before</summary>
<img width="1292" alt="Screen Shot 2021-04-23 at 9 06 44 PM" src="https://user-images.githubusercontent.com/1326294/115940428-ee3d7280-a477-11eb-933f-3a662c7a02fb.png">
<img width="1250" alt="Screen Shot 2021-04-23 at 9 06 05 PM" src="https://user-images.githubusercontent.com/1326294/115940431-f0073600-a477-11eb-9ab6-27f465569cba.png">
</details>
<details>
<summary>After</summary>
<img width="1219" alt="Screen Shot 2021-05-03 at 9 16 37 AM" src="https://user-images.githubusercontent.com/1326294/116879087-97ae0200-abf6-11eb-8988-012c13c68897.png">
<img width="1286" alt="Screen Shot 2021-05-03 at 9 16 08 AM" src="https://user-images.githubusercontent.com/1326294/116879098-9a105c00-abf6-11eb-8836-c3c0263cd2e3.png">
</details>

### Test

1. Run Mac Electron app.
2. Ensure bar appears at the top with the app below it as usual.
3. Check other Electron builds to ensure they work as expected.

### Release

- Added title bar area to the Mac Electron app
- Updated Electron and Electron Builder dependencies
